### PR TITLE
Improve nested if

### PR DESCRIPTION
**Pull request by NamPNQ**
Short Description: 
Imporve nested if

### CODE REVIEW

**Comment by douglascamata** [comment]
merge conflicts

**Comment by NamPNQ** [comment]
@douglascamata Resolved

File: pokemongo_bot/__init__.py [hunk 1]
```diff
@@ -363,25 +363,10 @@ def get_inventory(self):
...
-                                continue
-                            if not 'count' in item['inventory_item_data'][
-                                    'item']:
-                                continue
-                            self.inventory.append(item['inventory_item_data'][
-                                'item'])
+        if response.get('responses', {}).get('GET_INVENTORY', {}).get('inventory_delta', {}).get('inventory_items'):
+            for item in response['responses']['GET_INVENTORY']['inventory_delta']['inventory_items']:
+                if {"item_id", "count"}.issubset(set(item.get('inventory_item_data', {}).get('item', {}).keys())):
+                    self.inventory.append(item['inventory_item_data']['item'])
```
Comment (douglascamata) [inline pokemongo_bot/__init__.py]: This block of code looks awful. 

Please, name the variables used, like: 

```
inventory_items = response.get('responses', {}).get('GET_INVENTORY', {}).get('inventory_delta', {}).get('inventory_items')

if inventory_items:
    for item in inventory_items:
        item_dict = item.get('inventory_item_data', {}).get('item', {})
        if 'item_id' in item_dict and 'count' in item_dict:
            self.inventory.append(item_dict)
```

**Comment by NamPNQ** [comment]
@douglascamata Updated

**Comment by BobbyWibowo** [comment]
@NamPNQ I'm getting these with the latest commit:
```
Traceback (most recent call last):
  File "C:\PokemonGo-Bot\pokecli.py", line 494, in <module>
    main()
  File "C:\PokemonGo-Bot\pokecli.py", line 62, in main
    bot.start()
  File "C:\PokemonGo-Bot\pokemongo_bot\__init__.py", line 54, in start
    self._setup_api()
  File "C:\PokemonGo-Bot\pokemongo_bot\__init__.py", line 289, in _setup_api
    self.update_inventory()
  File "C:\PokemonGo-Bot\pokemongo_bot\__init__.py", line 397, in update_inventory
    if {"item_id", "count"}.issubset(set(item_info.keys())):
AttributeError: 'NoneType' object has no attribute 'keys'
```

**Comment by NamPNQ** [comment]
Oops, my mistake, tks @migolo

**Pull request merged**